src/utils/__tests__/findFontFamily-test.js

```diff
@@ -0,0 +1,80 @@
+import test from "tape"
+import findFontFamily from "../findFontFamily"
+
+test("findFontFamily", t => {
+  t.deepEqual(findFontFamily("12pt/10pt sans-serif"), [{ name: "sans-serif", index: 1 }])
+  t.deepEqual(findFontFamily("bold italic 110% serif"), [{ name: "serif", index: 3 }])
+  t.deepEqual(findFontFamily("normal small-caps 12px/14px fantasy"), [{ name: "fantasy", index: 3 }])
+  t.deepEqual(
+    findFontFamily("italic bold 12px/30px Georgia, serif"),
+    [
+      { name: "Georgia", index: 3 },
```

Comment:
Why is it returning the index within the list instead of the index in the string?